lib/resultuploaders/alicloudoss/alicloudoss.rb

```diff
@@ -0,0 +1,119 @@
+# frozen_string_literal: true
+
+require 'aliyun/oss'
+require './lib/auxiliary/json_helper'
+
+# AutoHCK module
+module AutoHCK
+  # alicloudoss class
+  class Alicloudoss
+    CONFIG_JSON = 'lib/resultuploaders/alicloudoss/alicloudoss.json'
+
+    include Helper
+
+    attr_reader :url
+
+    def initialize(project)
+      @tag = project.engine_tag
+      @timestamp = project.timestamp
+      @logger = project.logger
+      puts "@logger is  #{@logger}"

```

Comment:
Looks like leave after some debugging, please remove it.

Reply:
ok